browser/sources.gni

```diff
@@ -4,9 +4,49 @@
 # You can obtain one at http://mozilla.org/MPL/2.0/.
 
 import("//brave/browser/importer/sources.gni")
+import("//brave/components/binance/browser/buildflags/buildflags.gni")
+import("//third_party/widevine/cdm/widevine.gni")
```

Comment:
This PR needs to include widevine.gni to use enable_widevine guard in line #45. But, it causes below import loop error.

```
FAILED: build.ninja 
../../buildtools/linux64/gn --root=../.. -q gen .
ERROR //build/config/chrome_build.gni is part of an import loop.
See //brave/browser/sources.gni:8:1: whence it was imported.
import("//third_party/widevine/cdm/widevine.gni")
^-----------------------------------------------
See //brave/build/config/brave_build.gni:6:1: whence it was imported.
import("//brave/browser/sources.gni")
^-----------------------------------
See //build/config/chrome_build.gni:4:1: whence it was imported.
import("//brave/build/config/brave_build.gni")
^--------------------------------------------
See //build/config/compiler/compiler.gni:5:1: whence it was imported.
import("//build/config/chrome_build.gni")
^---------------------------------------
See //BUILD.gn:11:1: whence it was imported.
import("//build/config/compiler/compiler.gni")
^--------------------------------------------
ninja: error: rebuilding 'build.ninja': subcommand failed
```